src/types/module.rs

```diff
@@ -93,16 +93,41 @@ impl PyModule {
     ///
     /// # Examples
     ///
+    /// Include a file at compile time by using [`std::include_str` macro][1]:
+    ///
     /// ```no_run
```

Comment:
`no_run` will still compile the test (and try to find the file), but not run it. It has to not be compiled.

```suggestion
    /// ```ignore
```